internal/ethapi/api.go

```diff
@@ -942,13 +966,21 @@ func RPCMarshalBlock(b *types.Block, inclTx bool, fullTx bool) (map[string]inter
 		"miner":            head.Coinbase,
 		"difficulty":       (*hexutil.Big)(head.Difficulty),
 		"extraData":        hexutil.Bytes(head.Extra),
-		"size":             hexutil.Uint64(b.Size()),
+		"size":             hexutil.Uint64(head.Size()),
```

Comment:
This causes a change on the RPCMarchalBlock, because now it will return the header size, not the block size, right?

Reply:
Yes, thanks for catching it! I'll fix it tomorrow.